liquibase-standard/src/main/java/liquibase/analytics/configuration/AnalyticsArgs.java

```diff
@@ -99,7 +99,7 @@ public static boolean isAnalyticsEnabled() throws Exception {
         // if the user set enabled to false, that overrides all
         Boolean userSuppliedEnabled = ENABLED.getCurrentValue();
         if (Boolean.FALSE.equals(userSuppliedEnabled)) {
-            log.log(LOG_LEVEL.getCurrentValue(), "User has disabled analytics.", null);
+            log.log(Level.INFO, "User has disabled analytics.", null);
```

Comment:
The analytics log level is configurable. Is there a reason that you need this change to be made instead of configuring the log level to set it to info?

Reply:
Maybe it's my ignorance. How do I set the log level for this message via configuration (so that I don't have to change the code of Liquibase like I'm doing with this PR here)?

Reply:
The existing code gets the level to log at from the analytics log level parameter; all of the analytics logging is controlled by this parameter. This parameter is typically hidden from the help, but this is what you're looking for:

```
      --analytics-log-level=PARAM
                             DEFAULT: OFF
                             (defaults file: 'liquibase.analytics.logLevel',
                               environment variable:
                               'LIQUIBASE_ANALYTICS_LOG_LEVEL')
```